### PR TITLE
brig term layout improvements

### PR DESCRIPTION
**Pull request by willie-yao**
This PR contains a series of layout improvements for `brig term`

1. Move project name to be title of project box
2. Move event id to be title of event box
3. Make border color of event and job detail text boxes dependent on worker status on event page
4. Add event qualifiers and labels to the event info text box

### CODE REVIEW

**Comment by willie-yao** [comment]
image
I moved the event name to be the title of the event box, but since the top is split into two boxes: one for events and one for workers, it looks a bit off. Should I combine them under one larger box with a border? @krancour

**Comment by krancour** [comment]
> I moved the event name to be the title of the event box, but since the top is split into two boxes: one for events and one for workers, it looks a bit off. Should I combine them under one larger box with a border?

I think it's fine as is. The one thing I'd suggest is removing `Event: ` from the title of that box and only displaying the ID. I'd probably also make the color of the ID consistent with the color of the border.

Also, we can make it look less weird by making some modifications in the worker pane on the right. Let's move the word "Worker" to be the title of that textbox and _not_ prefix the other entries in that box with `Worker`.

File: v2/cli/term/event_page.go [hunk 1]
```diff
@@ -93,29 +102,47 @@ func (e *eventPage) refresh(eventID string) {
...
+	for k, v := range event.Qualifiers {
+		eventText = eventText + fmt.Sprintf("\n[yellow]%s: [white]%s", k, v)
+	}
+
+	// Add labels (if any) to event info box
+	for k, v := range event.Labels {
+		eventText = eventText + fmt.Sprintf("\n[yellow]%s: [white]%s", k, v)
+	}
+
+	e.eventInfo.SetText(eventText)
```
Comment (krancour) [inline v2/cli/term/event_page.go]: We need to differentiate between qualifiers and labels somehow.

Can we conditionally add "Qualifiers:" if there are any, then indent each of the qualifier lines?

Same for labels. Conditionally add "Labels:" if there are any, then indent each of the label lines.

File: v2/cli/term/project_page.go [hunk 2]
```diff
@@ -121,12 +121,11 @@ func (p *projectPage) refresh(projectID string) {
 
 func (p *projectPage) fillProjectInfo(project core.Project) {
 	p.projectInfo.Clear()
+	p.projectInfo.SetTitle(fmt.Sprintf("[yellow]Project: [white]%s\n", project.ID))
```
Comment (krancour) [inline v2/cli/term/project_page.go]: Looking at this in-use, I think it will look cleaner to omit `Project: `. Just use the ID as the title and that's it.

**Pull request merged**